packages/oasis-actions/src/helpers/swap/acceptedFeeToken.ts

```diff
@@ -30,6 +30,10 @@ export function acceptedFeeToken({ fromToken, toToken }: Props): 'sourceToken' |
     return fallbackTokenType
   }
 
+  if (fromTokenAcceptedIndex === -1) {
```

Comment:
Can you elaborate on this one ? What's the idea behind it.

Reply:
Have added some comments and improved variable names to try and make clearer